models/issue_comment.go

```diff
@@ -485,6 +491,25 @@ func createAssigneeComment(e *xorm.Session, doer *User, repo *Repository, issue
 	})
 }
 
+func createDeadlineComment(e *xorm.Session, doer *User, repo *Repository, issue *Issue, dateUnix util.TimeStamp, deadlineCommentType CommentType) (*Comment, error) {
```

Comment:
```
func createDeadlineComment(e *xorm.Session, doer *User, repo *Repository, issue *Issue, oldDeadline, newDeadline util.TimeStamp) (*Comment, error) {
	commentType := CommentTypeAddedDeadline
	deadline := newDeadline 

	if newDeadline == 0 {
		deadline = oldDeadline
		commentType := CommentTypeRemovedDeadline
	} else if oldDeadline != 0 {
		commentType := CommentTypeModifiedDeadline
	}

	content := deadline.Format("2006-01-02")

	return createComment(e, &CreateCommentOptions{
		Type:    commentType,
		Doer:    doer,
		Repo:    repo,
		Issue:   issue,
		Content: content,
	})
}
```

It would be great if content is more specific. When deadline is modified it should show both dates.

Reply:
Sure, but I rather won't add another row to the issue comment table as there are already too much. Someone said we should redo that some day, so I'll probably wait for that to do that...

Or do you have an idea on how to add both dates in one field?

Reply:
I meant if we can use `Content` and add both dates there.

Reply:
Yes, sure. I'll figure something out.

Reply:
Done.